src/pages/jdbc-create.html

```diff
@@ -9,205 +9,211 @@ <h1>
               popover-placement="bottom-right"
               popover-append-to-body="true"><span class="icon-info text-tertiary"></span></span>
     </h1>
-    <div core-errors></div>
+    <div core-errors license></div>
     <div system-repo-warning></div>
     <div class="alert alert-danger" ng-show="repositoryError">
         <p>{{'not.usable.active.repo.error' | translate}}</p>
 
         <p>{{repositoryError}}</p>
     </div>
+    <div ng-if="isLicenseValid()">
```

Comment:
As I can see, this in fact calls the isLicenseValid from the main controller. This is bad. Create a method in the local scope which delegates to the license service instead.